spark/src/main/java/org/apache/iceberg/spark/source/RowDataReader.java

```diff
@@ -150,10 +151,7 @@
               "Cannot read unknown format: " + task.file().format());
       }
     }
-
-    this.currentCloseable = iter;
-
-    return iter.iterator();
+    return iter;
```

Comment:
Here as well, it would be more readable to have a newline after the if/else block.

Reply:
OK